### DISCUSSION TITLE
alowing moderators to only list users information

### DISCUSSION THREAD

**Discussion by IsmaelAlvarez** [pocketbase/pocketbase#2613] [Q&A] [answered]
hi, i have 2 auth collections, users and moderators, but im kinda having trouble creating the list rule.

the idea is to allow users to list themselves like this i think? `id = @request.auth.id`
but for alowing the moderators to list all the users i might write something like: 
`@request.auth.id = "xxxxxxxxxxxx" || @request.auth.id = "xxxxxxxxxxxy" || ...`
but this is clearly not a good idea

the idea is to alow moderators to only list all the users, and not to manage the auth collection

**Comment by ganigeorgiev** [comment, marked as answer]
You can check the moderator collection name, something like:

```js
@request.auth.collectionName = "moderators" || id = @request.auth.id
```

**Reply by IsmaelAlvarez** [reply]
this works super, thanks!
